changelog/739.bugfix.rst

```diff
@@ -0,0 +1 @@
+Change :func:`ForumChannel.create_thread` to not require a ``content`` parameter.
```

Comment:
```suggestion
Change :func:`ForumChannel.create_thread` to not require the ``content`` parameter.
```

Reply:
```suggestion
Change :func:`ForumChannel.create_thread` to not require the ``content`` parameter to be provided.
```
like that maybe?

Reply:
Hm, we should probably allude to all of the required parameters in some form.

Reply:
https://github.com/DisnakeDev/disnake/pull/739/commits/8b510731db958ee71c6bee601be9da64ecd3fbd7